development-docs/TESTING.md

```diff
@@ -34,10 +34,7 @@ You can find tests inside `src.test.java` package. Moreover we have the auxiliar
 - `Clients` clients for testing overall communication of the `Kafka Bridge`
 - `Facades` encapsulation of the standalone `Kafka` and `AdminClient` instance
 
-Furthermore we divide our test suite on two separate parts:
-
-1. `Http`package, where you can find all test cases related to the `Http Kafka Bridge`
-2. `Amqp`package, where you can find all tests cases related to the `Amqp Kafka Bridge`
+The test suite has provides a `Http`package, where you can find all test cases related to the `Http Kafka Bridge`.
```

Comment:
Is the package really `Http`? Or is it `http`? Package names are normally lowercase.

```suggestion
The test suite has provides a `Http` package, where you can find all test cases related to the `Http Kafka Bridge`.
```